tests/unit/test-cmd/test.create.js

```diff
@@ -0,0 +1,118 @@
+/* @flow */
+import path from 'path';
+import tty from 'tty';
+
+import {fs} from 'mz';
+import {describe, it, afterEach} from 'mocha';
+import {assert} from 'chai';
+import sinon from 'sinon';
+import mockStdin from 'mock-stdin';
+
+import create from '../../../src/cmd/create';
+import {withTempDir} from '../../../src/util/temp-dir';
+import {makeSureItFails} from '../helpers';
+import {onlyInstancesOf, UsageError} from '../../../src/errors';
+
+const homeDir = process.cwd();
+
+describe('create', () => {
+
+  afterEach(() => {
+    process.chdir(homeDir);
+  });
+
+  it('creates files including manifest with correct name', () => withTempDir(
+    (tmpDir) => {
+      process.chdir(tmpDir.path());
```

Comment:
Like discussed, we should change the current work dir during the tests (and it should fix the failures of these tests on windows) here, in the afterEach and elsewhere in this test file.